docs/Windows.md

```diff
@@ -38,7 +38,117 @@ Additionally, the ICU headers and libraries need to be provided for the build.
+
+### 2. Clone the repositories
+1. Create a folder to contain all the Swift repositories
+2. `apple/swift-cmark` into a folder named `cmark`
+3. `apple/swift-clang` into a folder named `clang`
+5. `apple/swift-llvm` into a folder named `llvm`
+5. `apple/swift` into a folder named `swift`
+- Currently, other repositories in the Swift project have not been tested, and may not be supported.
+
+### 3. Build ICU
+1. Download and extract the [ICU source code](http://site.icu-project.org/download) to a folder named `icu` in the same directory as the other Swift project repositories.
+2. Open `src/win32/allinone.sln` in Visual Studio.
+3. Make sure to select the correct architecture from the drop-down in Visual Studio.
+4. Right click on the solution in the Solution Explorer window and select `Build Solution`.
+5. When this is done, add the `<icu-source>/bin` folder to your `Path` environment variable.
+
+### 4. Get ready
+- From within a **developer** command prompt, execute the following command if you have an x64 PC.
+```
+"C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/vcvarsall.bat" amd64
+```
+- Then adapt the following command, and run it.
+```
+set swift_source_dir=path-to-directory-containing-all-cloned-repositories
+```
+
+### 5. Build CMark
+- This must be done from within a developer command prompt, and could take up to 10 minutes.
+```
+mkdir "%swift_source_dir%/build/Ninja-DebugAssert/cmark-windows-amd64"
+pushd "%swift_source_dir%/build/Ninja-DebugAssert/cmark-windows-amd64"
+cmake -G "Ninja" "%swift_source_dir%/cmark"
+popd
+cmake --build "%swift_source_dir%/build/Ninja-DebugAssert/cmark-windows-amd64/"
+```
+
+### 6. Build LLVM/Clang/Compiler-RT
+- This must be done from within a developer command prompt, and could take up to 5 hours.
+```
+mklink /J "%swift_source_dir%/llvm/tools/clang" "%swift_source_dir%my-swift/clang"
+mklink /J "%swift_source_dir%/llvm/tools/compiler-rt" "%swift_source_dir%/compiler-rt"
+mkdir "%swift_source_dir%/build/Ninja-DebugAssert/llvm-windows-amd64"
+pushd "%swift_source_dir%/build/Ninja-DebugAssert/llvm-windows-amd64"
+cmake -G "Ninja"^
+ -DLLVM_ENABLE_ASSERTIONS=TRUE^
+ -DCMAKE_BUILD_TYPE=Debug^
+ -DLLVM_TOOL_SWIFT_BUILD=NO^
+ -DLLVM_INCLUDE_DOCS=TRUE^
+ -DLLVM_TOOL_COMPILER_RT_BUILD=TRUE^
+ -DLLVM_BUILD_EXTERNAL_COMPILER_RT=TRUE^
+ -DLLVM_LIT_ARGS=-sv^
+ "%swift_source_dir%/llvm"
+popd
+cmake --build "%swift_source_dir%/build/Ninja-DebugAssert/llvm-windows-amd64"
+```
+
+### 7. Build Swift
+- This must be done from within a developer command prompt, and could take up to 2 hours.
+- You may need to adjust the SWIFT_WINDOWS_LIB_DIRECTORY parameter depending on your target platform or Windows SDK version.
+```
+mkdir "%swift_source_dir%/build/Ninja-DebugAssert/swift-windows-amd64/ninja"
+pushd "%swift_source_dir%/build/Ninja-DebugAssert/swift-windows-amd64/ninja"
+cmake -G "Ninja" "%swift_source_dir%/swift"^
+ -DCMAKE_CXX_FLAGS="/std:c++14"^
```

Comment:
I've removed this, and also got rid of the mentions of libedit (the cmake flag to disable it no longer exists)